packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -7,20 +7,30 @@ import {
     PaymentIntegrationService,
     PaymentRequestOptions,
     PaymentStrategy,
+    TimeoutError,
 } from '@bigcommerce/checkout-sdk/payment-integration-api';
 
 import PayPalCommerceIntegrationService from '../paypal-commerce-integration-service';
 
 import { WithPayPalCommerceAlternativeMethodsPaymentInitializeOptions } from '../index';
-import { BirthDate, PayPalCommerceInitializationData } from '../paypal-commerce-types';
+import {
+    BirthDate,
+    PayPalCommerceInitializationData,
+    PayPalOrderStatus,
+} from '../paypal-commerce-types';
 import { PaypalCommerceRatePay } from './paypal-commerce-alternative-methods-payment-initialize-options';
+import { noop } from 'lodash';
 
 export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStrategy {
     private guid?: string;
     private paypalcommerceratepay?: PaypalCommerceRatePay;
+    private pollingTimer = 0;
+    private stopPolling = noop;
     constructor(
         private paymentIntegrationService: PaymentIntegrationService,
         private paypalCommerceIntegrationService: PayPalCommerceIntegrationService,
+        private pollingInterval = 3000,
+        private maxPollingTime = 600000,
```

Comment:
Why did you place this variables into constructor props?

Reply:
Removed from constructor, replaced with constants